src/charm.py

```diff
@@ -29,13 +29,13 @@ def __init__(self, *args):
         super().__init__(*args)
         self._container_name = "login_ui"
         self._container = self.unit.get_container(self._container_name)
-        """New vars should come here"""
+        self._application_port = "8080"
```

Comment:
could you make it a constant, like [here](https://github.com/canonical/hydra-operator/blob/main/src/charm.py#L41)?